x-pack/plugins/secops/public/components/page/hosts/host_summary/index.test.tsx

```diff
@@ -95,14 +107,16 @@ describe('Host Summary Component', () => {
 
   describe('#getEuiDescriptionList', () => {
     test('if it creates a description list', () => {
-      const euiDescriptionList = getEuiDescriptionList(
-        mockData.Hosts.edges[0].node,
-        552204000000,
-        618472800000
-      );
+      const mockNode: HostItem = mockData.Hosts.edges[0].node;
+      mockNode.firstSeen = moment()
```

Comment:
I would avoid basing a test on date time now such as a subtraction. Is it possible to use an absolute date time again?